test/TestServerProjectsLowLevel/dpg-customization/Models/LROProduct.Serialization.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Do we need LRO models if we're not adding LRO tests yet?  It might be better to wait to add them until we're using them.